package.json

```diff
@@ -21,12 +21,15 @@
     "trailingComma": "all"
   },
   "dependencies": {
+    "bcrypt": "^5.0.1",
```

Comment:
No idea why bcrypt was added here. If anything it should've been used with `register` method to encrypt user's password.

Reply:
remove from login branch